samples/msgext-ai-sentiment-analysis/nodejs/Readme.md

```diff
@@ -0,0 +1,124 @@
+---
+page_type: sample
+description: This sample shows a feature where user can check sentiment for teams message by using messaging extension.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "08/07/2023 04:00:00 PM"
+urlFragment: officedev-microsoft-teams-samples-msgext-ai-sentiment-analysis-nodejs
+---
+
+# Sentiment Analysis with messaging extension
+
+This sample shows a feature where user can check sentiment for teams message by using messaging extension.
+
+## Included Features
+* Bots
+* Open AI
```

Comment:
Azure Open AI

Please use this at all the places where you mentioned Open AI.

Reply:
fixed